mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,17 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrderById = (req, res) => {
+  Order.deleteOne({ _id: req.params.orderId })
```

Comment:
The code snippet provided is part of the `removeOrderById` function in the `order.js` controller. 

Here is the review:

In the `removeOrderById` function, the `Order.deleteOne` method is used to delete an order by its ID. However, there is no error handling for the case where the order ID is not found. 

It would be better to handle this scenario by checking the result of the `deleteOne` method and returning a 404 status code if the order is not found. This is already done in the code, but it's good to mention it in the review.

Additionally, the function name `removeOrderById` is not following the conventional naming convention used in the codebase. It should be camelCase, i.e., `removeOrderById` should be `removeOrderId`.